crates/project-model/src/cargo_workspace.rs

```diff
@@ -297,6 +297,7 @@ impl CargoWorkspace {
             let other_options: Vec<_> = targets
                 .into_iter()
                 .flat_map(|target| ["--filter-platform".to_string(), target])
+                .chain(config.extra_args.clone())
```

Comment:
This is only set when the targets are empty here, we should put this after the `!targets.is_empty()` branch

Reply:
Should be fixed now